numba/tests/test_np_randomgen.py

```diff
@@ -231,9 +232,11 @@ def test_npgen_boxing_refcount(self):
         do_box = numba.njit(lambda x:x)
 
         y = do_box(rng_instance)
+        gc.collect()
```

Comment:
Could this use `with self.assertRefCount(...):` ?

Reply:
it's expecting a `+ 1` so it cannot use that testmethod

Reply:
ah ok, thanks for checking.